cmd/user-agent/src/pages/DIDManagement.vue

```diff
@@ -80,10 +80,23 @@ SPDX-License-Identifier: Apache-2.0
                                     </md-field>
                                 </div>
                                 <div class="md-layout-item md-size-100">
-                                    <md-icon>memory</md-icon> <select id="selectSignKey" v-model="signType" style="color: grey; width: 300px; height: 35px;">
-                                    <option value="" disabled="disabled">Select Signature Type</option>
+                                    <md-field maxlength="5">
+                                        <label class="md-helper-text">Type DID friendly name here</label>
+                                        <md-input v-model="anyDIDFriendlyName" id="anyDIDFriendlyName" required></md-input>
+                                    </md-field>
+                                </div>
+                                <div class="md-layout-item md-size-100">
+                                    <md-icon>memory</md-icon> <select id="privateKeyType" v-model="privateKeyType" style="color: grey; width: 200px; height: 35px;">
+                                    <option value="">Select Key Type</option>
+                                    <option value="Ed25519">Ed25519</option>
+                                </select>
+                                    <md-field style="margin-top: -15px">
+                                    </md-field>
+                                </div>
+                                <div class="md-layout-item md-size-100">
+                                    <md-icon>memory</md-icon> <select id="selectSignKey" v-model="selectSignKey" style="color: grey; width: 300px; height: 35px;">
+                                    <option value="">Select Signature Type</option>
                                     <option value="Ed25519Signature2018">Ed25519Signature2018</option>
-                                    <option value="JsonWebSignature2020">JsonWebSignature2020</option>
```

Comment:
Why is this being removed?

Reply:
vendors DID don't support JsonWebSignature2020

Reply:
when they support will add it